doc/elements.md

```diff
@@ -216,6 +216,23 @@ def file(data):
 
 If `file()` does not return anything, it will be treated as if `file()` returned the empty string.
 
+## `pl-prairiedraw-figure` element
+
+```html
+<pl_prairiedraw_figure script_name="drawFigure.js" params_names="r1,r2,isHorizontal" width="900" height="400" />
+```
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`script-name` | string | - | Name of PrairieDraw script.
+`params-names` | string | `None` | Comma-separated list of parameters to make available to PrairieDraw.
```

Comment:
Nit: `param-names` seems to me to be a better name for this and is more consistent with attributes like `file-names` in `pl-file-upload`.

Reply:
I agree with your reasoning on the change; my concern is backwards compatibility. The TAM 2XX classes have used this element (course-level) for several semesters, so changing this attribute name will break a large collection of existing problems.

Does it make sense to tweak the element code to search for `param-names` and then `params-names` so older questions work without changes, but let the documentation reflect the new name?

Reply:
You could do that, but I would also take this as a good opportunity to make breaking changes. This is effectively a brand-new element; courses that already use it have to explicitly opt-in to this version by deleting the copy from their course. A part of that process could be updating the usage; thankfully, this change is probably trivial to do with find/replace.

Reply:
That seems like a reasonable solution; since everyone using this currently has a course-level version of the element, nothing existing should break (I hope).

Reply:
You can verify it with your local course + PL if you have concerns about it breaking :)